bench.json

```diff
@@ -4,9 +4,9 @@
     "run": "esy '@bench' x TextMateBench"
   },
   "override": {
-      "build": ["dune build --root . -j4"],
+      "build": ["dune build @all -j4"],
```

Comment:
I tried replacing `dune build --root . -j4` with `dune build -p textmate,textmate-bench -j4` and it seems to work locally. This is better anyway since it doesn't need to build the test project for bench, and vice versa.

Reply:
What's the reason you want back to `-p textmate,textmate-bench`?